contracts/child/misc/ParentTokenMock.sol

```diff
@@ -1,19 +0,0 @@
-pragma solidity ^0.5.2;
-
-import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
-
-import "./IParentToken.sol";
-// demo token parent contract  
-
-
-contract ParentTokenMock is IParentToken, Ownable {
```

Comment:
please add it back, it's used to test `IParentToken` interface and also a sample implementation of it.
feel free to rename it.

Reply:
Added back.